### DISCUSSION TITLE
Flashing FW on MacOS ???

### DISCUSSION THREAD

**Discussion by ghhausi** [john30/ebusd#1115] [Q&A]
Hi,
I need help for **flashing** the adapter firmware on a **MacOS system**.
I have a Hackintosh and a MacBookPro - and Chrome. 
Did connect the adapter via USB (and hold the button on power-on - the blue led is on then) - but could **not see anything on Chrome to connect**. Did not see any /dev/cu*serial* ..... did not see anything like "JTAG/serial" ....
Do I have to install something ?

Q: Who did that on Mac-systems successfully ? Can you provide a "guide" or steps to follow ?
     How do you guys flash the firmware ?
     (I have no Windows system - just Mac and Linux)

Thanks
Gerhard

**Comment by SzosszeNET** [comment]
Same boot, in addition I have apple silicon hence unable to install the drivers.

On Hackintosh however should be able to install the drivers as you have intel cpu?

**Reply by ghhausi** [reply]
Thanks,
I managed to update the FW on my Linux system ....
